Formula/rbw.rb

```diff
@@ -0,0 +1,19 @@
+class Rbw < Formula
+  desc "Unoffical Bitwarden CLI client"
+  homepage "https://github.com/doy/rbw"
+  url "https://github.com/doy/rbw/archive/refs/tags/1.3.0.tar.gz"
+  sha256 "7255e322b8c6bd6ee9da00f4c54211e94a132abec15b9f51b8351af31a4744ac"
+  license "MIT"
+  head "#{homepage}.git", branch: "master"
+
+  depends_on "rust" => :build
+  depends_on "pinentry"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "rbw", "gen-completions", "bash"
```

Comment:
Can we do a test that tries to use some real functionality of `rbw`?

Reply:
`rbw` requires an active Bitwarden account to use any functionality other than the one above. The only other viable test I can think of for something like this would be to pull the tarball again and invoke `cargo test` instead of `cargo install`.

Reply:
How about checking that it complains about a missing bitwarden account?

Reply:
Sorry for the late reply. The output produced by a missing Bitwarden account prints out the user's directory in its "file not found" error output near the end, though this could be stripped out.